src/OrbitGl/CaptureViewElementTester.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_CAPTURE_VIEW_ELEMENT_TESTER_H_
+#define ORBIT_GL_CAPTURE_VIEW_ELEMENT_TESTER_H_
+
+#include "CaptureViewElement.h"
+
+namespace orbit_gl {
+
+class CaptureViewElementTester {
```

Comment:
Nit: Please add a brief comment about this class here.
See https://google.github.io/styleguide/cppguide.html#Class_Comments